examples/sample-maven-project/src/main/java/edu/stanford/nlp/StanfordCoreNLPChineseTestApp.java

```diff
@@ -1,20 +1,23 @@
 package edu.stanford.nlp;
 
-import java.io.*;
 import java.util.*;
-import edu.stanford.nlp.io.*;
+import edu.stanford.nlp.ling.CoreLabel;
 import edu.stanford.nlp.pipeline.*;
-import edu.stanford.nlp.util.*;
 
-/** app for testing if Maven distribution is working properly */
+/**
+ * Application to test whether the new version of Stanford works properly
+ */
+public class StanfordCoreNLPChineseTestApp {
 
-public class StanfordCoreNLPChineseTestApp
-{
-    public static void main(String[] args) throws IOException, ClassNotFoundException
-    {
-        String[] chineseArgs = new String[]{"-props","StanfordCoreNLP-chinese.properties",
-                                              "-annotators","tokenize,ssplit,pos,lemma,ner,parse,coref,kbp,entitylink",
-                                              "-file", "sample-chinese.txt", "-outputFormat", "text"};
-        StanfordCoreNLP.main(chineseArgs);
+    public static void main(String[] args) {
+        Properties properties = new Properties();
+        properties.setProperty("annotators", "tokenize,ssplit,pos,lemma,ner,parse,coref,kbp,entitylink");
```

Comment:
Chinese properties for the Chinese test app?

Reply:
I still don't think this is it... the Chinese properties are not being loaded and the "annotators" field is overwritten right away.  What's the purpose for this update?  Is the original version not working?